content/en/actions/agents/_index.md

```diff
@@ -0,0 +1,150 @@
+
+To build an agent from a blueprint:
+
+1. Navigate to the [Agents][1] page. 
+1. In **Agent Blueprints**, click the blueprint you want to use. 
+    1. Alternatively, you can click **New Agent** and browse the full list of blueprints.
+1. Click **Create From Blueprint**.
+1. Your new agent will have pre-populated settings and tools. You can immediately start chatting with the agent or [further customize it](#customize-your-agent). 
+1. Click the gear icon to close the agent's settings.
+
+### Create a custom agent
+
+To build an agent from scratch: 
+
+1. Navigate to the [Agents][1] page. 
+1. Click **New Agent**. 
+1. [Customize your agent](#customize-your-agent). Changes are saved automatically.
+1. Start chatting with your agent. 
+
+## Customize your agent
+
+When configuring your agent, you have the following options for customization. 
+
+### Model
+
+Click the dropdown menu to choose your GPT model. Current options include [GPT-4o][3], [GPT-4.1][4], and [GPT-5][5]. You can compare these models using [OpenAI's tool comparison tool][6].
+
+### Instructions
+
+Enter the instructions your agent will follow when performing tasks. You can describe the agent as if it were a person, including its role, specialties, and steps to follow. Click **Suggest With AI** to get help with writing and refining your instructions. 
+
+Below is a sample set of instructions for an agent acting as an incident responder.
+
+```
+You are an Incident Responder AI assistant specialized in managing and coordinating incident response activities.
+
+Your role involves:
+- Guiding incident response procedures and best practices
+- Helping assess incident severity and impact
+- Coordinating communication between teams and stakeholders
+- Managing incident lifecycle from detection to resolution
+- Facilitating post-incident reviews and improvements
+
+During incident response:
+1. Help classify incident severity (P0/P1/P2/etc.)
+2. Guide through incident response runbooks and procedures
+3. Assist with stakeholder communication and updates
+4. Track action items and follow-up tasks
+5. Support post-mortem analysis and lessons learned
+
+Focus on clear communication, structured processes, and continuous improvement of incident response capabilities.
+```
+
+### Tools
+
+Click **+ Add Tool** to add actions from our [Action Catalog][7] for the agent to use. The agent is restricted to only its added tools. For example, if you encounter an error when the agent attempts to perform a task you've requested, it might be missing a necessary action. 
+
+### MCP Servers
+
+Click the toggle button to enable [MCP (Model Context Protocol)][8]. MCP servers provide additional tools and capabilities that the agent can use to interact with external systems.
```

Comment:
"MCP" is just a list of specifications; the `Datadog MCP Server` is the thing users can actually use.
```suggestion
Click the toggle button to enable the [Datadog MCP Server][8]. MCP servers use the Model Context Protocol to provide additional tools and capabilities that the agent can use to interact with external systems.
```